test/collection-capabilities.ts

```diff
@@ -1,5 +1,6 @@
-import { expect } from 'chai';
+import _ from 'lodash';
 import { currentApiVersion } from '../app/frontends/capabilities';
+import { expect } from 'chai';
```

Comment:
nit: It looks like you are sorting the imports in alphabetical order (based on the thing imported, not the module name?). The loose convention we seem to have is to list npm modules at the top and our own modules below those. So 'chai' would come before '.../capabilities'. Not a big deal and I don't really care which convention we use, we should just be consistent.

Reply:
Good point. I will update the imports based on harmony convention.